MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -880,20 +880,35 @@ - (void)computeDrift {
     float playheadTimeElapsed = ([self getCurrentPlayheadTimeMs] - _lastPlayheadTimeMs) / 1000;
     float wallTimeElapsed = CFAbsoluteTimeGetCurrent() - _lastPlayheadTimeUpdated;
     float drift = playheadTimeElapsed - wallTimeElapsed;
+    
     // The playhead has to have moved > 500ms and we have to have signifigantly drifted in comparision to wall time.
     // We check both positive and negative to account for seeking forward and backward respectively.
     // Unbuffered seeks seem to update the playhead time when transitioning into play where as buffered seeks update the playhead time when paused.
     if (fabsf(playheadTimeElapsed) > MUXSDKMaxSecsSeekPlayheadShift &&
-        fabsf(drift) > MUXSDKMaxSecsSeekClockDrift &&
-        (_state == MUXSDKPlayerStatePaused || _state == MUXSDKPlayerStatePlay)) {
-        _seeking = YES;
-        MUXSDKInternalSeekingEvent *event = [[MUXSDKInternalSeekingEvent alloc] init];
-        MUXSDKPlayerData *playerData = [self getPlayerData];
-        if ([[UIDevice currentDevice] userInterfaceIdiom] == UIUserInterfaceIdiomTV) {
-            [self setPlayerPlayheadTime:_lastPlayheadTimeMsOnPause onPlayerData:playerData];
+        fabsf(drift) > MUXSDKMaxSecsSeekClockDrift) {
+        if (_state == MUXSDKPlayerStatePaused || _state == MUXSDKPlayerStatePlay) {
+            _seeking = YES;
+            MUXSDKInternalSeekingEvent *event = [[MUXSDKInternalSeekingEvent alloc] init];
+            MUXSDKPlayerData *playerData = [self getPlayerData];
+            if ([[UIDevice currentDevice] userInterfaceIdiom] == UIUserInterfaceIdiomTV) {
+                [self setPlayerPlayheadTime:_lastPlayheadTimeMsOnPause onPlayerData:playerData];
+            }
+            [event setPlayerData:playerData];
+            [MUXSDKCore dispatchEvent:event forPlayer:_name];
+        } else if (_state == MUXSDKPlayerStatePlaying) {
+            // Programatically called seek
```

Comment:
couldn't the system programatically call seek if the player was paused as well? this comment implies that the state is playing for any programatic seek

Reply:
Yes it could be called on pause my intention with this comment was more oriented to the fact that this block of code would only enter if seek was called programatically.

Reply:
Thanks for explaining this - could we expand this comment a bit to say `when programmatically calling seek, it's possible the player never goes through the paused state` so that it is clear you could also go through the top branch if seeking programmatically

Reply:
Yeah I can expand